src/components/Drawer/MiniSideBar.tsx

```diff
@@ -0,0 +1,163 @@
+import * as React from "react";
+import Box from "@mui/material/Box";
+import Toolbar from "@mui/material/Toolbar";
+import CssBaseline from "@mui/material/CssBaseline";
+import Divider from "@mui/material/Divider";
+import { Button, Drawer, Typography, SvgIcon } from "@mui/material";
+import { PropsWithChildren } from "react";
+import CampaignOutlinedIcon from "@mui/icons-material/CampaignOutlined";
+import AccountBalanceOutlinedIcon from "@mui/icons-material/AccountBalanceOutlined";
+import AccountBoxOutlinedIcon from "@mui/icons-material/AccountBoxOutlined";
+import HeadsetMicOutlinedIcon from "@mui/icons-material/HeadsetMicOutlined";
+import InsertPhotoOutlinedIcon from "@mui/icons-material/InsertPhotoOutlined";
+import LightbulbOutlinedIcon from "@mui/icons-material/LightbulbOutlined";
+
+type RouteOption = {
+  label: string;
+  href: string;
+  icon: React.ReactNode;
+  disabled?: boolean;
+};
+
+const drawerWidth = 120;
+export default function MiniSideBar({ children }: PropsWithChildren) {
+  const dashboardRoutes: RouteOption[] = [
+    {
+      label: "Campaigns",
+      href: "/user/main/campaign",
+      icon: (
+        <CampaignOutlinedIcon
+          fontSize="large"
+          sx={{ color: "rgba(161, 171, 186, 1)" }}
+        />
+      ),
+    },
+    // Possible future enhancements, not visible to user but help keep spacing
+    {
+      label: "Creatives",
+      href: "/user/main/creatives",
+      icon: (
+        <LightbulbOutlinedIcon
+          fontSize="large"
+          sx={{ color: "rgba(161, 171, 186, 1)" }}
+        />
+      ),
+      disabled: true,
+    },
+    {
+      label: "Assets",
+      href: "/user/main/assets",
+      icon: (
+        <InsertPhotoOutlinedIcon
+          fontSize="large"
+          sx={{ color: "rgba(161, 171, 186, 1)" }}
+        />
+      ),
+      disabled: true,
+    },
+    {
+      label: "Audiences",
+      href: "/user/main/audiences",
+      icon: (
+        <CampaignOutlinedIcon
+          fontSize="large"
+          sx={{ color: "rgba(161, 171, 186, 1)" }}
+        />
+      ),
+      disabled: true,
+    },
+  ];
+
+  const settingsRoutes: RouteOption[] = [
+    {
+      label: "Account",
+      href: "/user/main/settings",
+      icon: (
+        <AccountBalanceOutlinedIcon
+          fontSize="large"
+          sx={{ color: "rgba(161, 171, 186, 1)" }}
+        />
+      ),
+    },
+    {
+      label: "Profile",
+      href: "/user/main/profile",
+      icon: (
+        <AccountBoxOutlinedIcon
+          fontSize="large"
+          sx={{ color: "rgba(161, 171, 186, 1)" }}
+        />
+      ),
+    },
+    {
+      label: "Support",
+      href: "mailto:selfserve@brave.com",
+      icon: (
+        <HeadsetMicOutlinedIcon
+          fontSize="large"
+          sx={{ color: "rgba(161, 171, 186, 1)" }}
+        />
+      ),
```

Comment:
It's a bit weird that clicking this sidebar icon exits from the app and opens gmail (for me). Could it open a full page screen instead with details of how to get support?